src/options.cpp

```diff
@@ -1320,6 +1322,31 @@ void show_options(bool ingame)
             g->init_ui();
             if( ingame ) {
                 g->refresh_all();
+
+                DebugLog( D_INFO, DC_ALL ) << "Loaded tileset: " << OPTIONS["TILES"].getValue();
+
+                tile_loading_report(termap, "Terrain", "");
+                tile_loading_report(furnmap, "Furniture", "");
+                tile_loading_report(Item_factory::Item_factory().get_all_itypes(), "Items", "");
+                tile_loading_report(MonsterGenerator::generator().get_all_mtypes(), "Monsters", "");
+                tile_loading_report(vehicle_part_types, "Vehicle Parts", "vp_");
+                tile_loading_report(trapmap, "Traps", "");
+
+                // fields are the only tile-able thing not kept in a map!
+                int missing=0, present=0;
+                std::string missing_list;
+                for(int i = 0; i < num_fields; ++i) {
+                    if (tilecontext->get_tile_ids().count(fieldlist[i].id) == 0) {
```

Comment:
Same here: a copy of the loaded tiles for every field. Do you *want* this code to be slow?